.editorconfig

```diff
@@ -7,3 +7,4 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
 indent_size = 2
+use_tabs = false
```

Comment:
What's the purpose of `use_tabs`? I don't see such config documented in [editorconfig docs](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties).

We already have `indent_style = space`.